Tests fail because of missing module 'jsonschema'

```
$ python3 -m pytest --cov=clkhash
=========================== test session starts ===========================
platform linux -- Python 3.6.5rc1, pytest-3.4.2, py-1.5.2, pluggy-0.6.0
rootdir: /home/hlaw/src/n1-clkhash, inifile:
plugins: cov-2.5.1
collected 76 items / 1 errors                                             


--------- coverage: platform linux, python 3.6.5-candidate-1 ---------
Name                          Stmts   Miss  Cover
-------------------------------------------------
clkhash/__init__.py               9      2    78%
clkhash/benchmark.py             27     18    33%
clkhash/bloomfilter.py           93     66    29%
clkhash/cli.py                  120     65    46%
clkhash/clk.py                   65     47    28%
clkhash/identifier_types.py      24     12    50%
clkhash/key_derivation.py        45     33    27%
clkhash/randomnames.py           52     34    35%
clkhash/schema.py                19     14    26%
clkhash/stats.py                 29     21    28%
clkhash/tokenizer.py             16     12    25%
-------------------------------------------------
TOTAL                           499    324    35%

================================= ERRORS ==================================
_______________ ERROR collecting tests/test_json_schema.py ________________
ImportError while importing test module '/home/hlaw/src/n1-clkhash/tests/test_json_schema.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_json_schema.py:5: in <module>
    from jsonschema import validate
E   ModuleNotFoundError: No module named 'jsonschema'
!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!
========================= 1 error in 0.40 seconds =========================
```

--- a/README.md
+++ b/README.md
@@ -70,7 +70,15 @@ As a rule of thumb a single modern core will hash around 1M entities in about 20
 
 # Tests
 
-Run unit tests and print out code coverage with `py.test`:
+Make sure you have all the required modules before running the tests
+(modules that are only needed for tests are not included during
+installation):
+
+```
+pip install -r requirements.txt
+```
+
+Now run the unit tests and print out code coverage with `py.test`:
 
 ```
 $ python -m pytest --cov=clkhash

